Traceback (most recent call last):
  File "/Users/patrickwalls/anaconda3/envs/jb/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/patrickwalls/anaconda3/envs/jb/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/patrickwalls/anaconda3/envs/jb/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/Users/patrickwalls/anaconda3/envs/jb/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/Users/patrickwalls/anaconda3/envs/jb/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/Users/patrickwalls/anaconda3/envs/jb/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/patrickwalls/anaconda3/envs/jb/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
x = [1557,1452,767,900,1018,802,924,981,879,819,829,1088,1085]
y = [1.398,1.750,0.899,0.848,1.149,0.825,0.999888,1.175,0.885,0.749888,1.098,1.099,1.198]
plt.plot(x,y,'.')
plt.title('Listing Price on REALTOR.ca')
plt.xlabel('Apartment Size (sqft)'), plt.ylabel('Price (Millons $)'), plt.grif(True)
plt.show()
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mAttributeError[0m                            Traceback (most recent call last)
Cell [0;32mIn[2], line 5[0m
[1;32m      3[0m plt[38;5;241m.[39mplot(x,y,[38;5;124m'[39m[38;5;124m.[39m[38;5;124m'[39m)
[1;32m      4[0m plt[38;5;241m.[39mtitle([38;5;124m'[39m[38;5;124mListing Price on REALTOR.ca[39m[38;5;124m'[39m)
[0;32m----> 5[0m plt[38;5;241m.[39mxlabel([38;5;124m'[39m[38;5;124mApartment Size (sqft)[39m[38;5;124m'[39m), plt[38;5;241m.[39mylabel([38;5;124m'[39m[38;5;124mPrice (Millons $)[39m[38;5;124m'[39m), [43mplt[49m[38;5;241;43m.[39;49m[43mgrif[49m([38;5;28;01mTrue[39;00m)
[1;32m      6[0m plt[38;5;241m.[39mshow()

[0;31mAttributeError[0m: module 'matplotlib.pyplot' has no attribute 'grif'

